addons/l10n_ro_edi_stock/models/l10n_ro_edi_stock_etransport_interface.py

```diff
@@ -0,0 +1,640 @@
+    ('162801', "BVI Craiova (ROCR2100)"),
+    ('162901', "BVF Craiova Aeroport (ROCR2110)"),
+    ('162902', "BVF Bechet (ROCR1720)"),
+    ('162903', "BVF Calafat (ROCR1700)"),
+    ('172901', "BVF Zona Liberă Galaţi (ROGL3810)"),
+    ('172902', "BVF Giurgiuleşti (ROGL3850)"),
+    ('172903', "BVF Oancea (ROGL3610)"),
+    ('172904', "BVF Galaţi (ROGL3800)"),
+    ('182801', "BVI Târgu Jiu (ROCR8810)"),
+    ('192801', "BVI Miercurea Ciuc (ROBV5600)"),
+    ('202801', "BVI Deva (ROTM8100)"),
+    ('212801', "BVI Slobozia (ROCT8220)"),
+    ('222901', "BVF Iaşi Aero (ROIS4660)"),
+    ('222902', "BVF Sculeni (ROIS4990)"),
+    ('222903', "BVF Iaşi (ROIS4650)"),
+    ('232801', "BVI Antrepozite/Ilfov (ROBU1200)"),
+    ('232901', "BVF Otopeni Călători (ROBU1030)"),
+    ('242801', "BVI Baia Mare (ROCJ0500)"),
+    ('242901', "BVF Aero Baia Mare (ROCJ0510)"),
+    ('242902', "BVF Sighet (ROCJ8000)"),
+    ('252901', "BVF Orşova (ROCR7280)"),
+    ('252902', "BVF Porţile De Fier I (ROCR7270)"),
+    ('252903', "BVF Porţile De Fier II (ROCR7200)"),
+    ('252904', "BVF Drobeta Turnu Severin (ROCR9000)"),
+    ('262801', "BVI Târgu Mureş (ROBV8800)"),
+    ('262901', "BVF Târgu Mureş Aeroport (ROBV8820)"),
+    ('272801', "BVI Piatra Neamţ (ROIS7400)"),
+    ('282801', "BVI Corabia (ROCR2000)"),
+    ('282802', "BVI Olt (ROCR8210)"),
+    ('292801', "BVI Ploiești (ROBU7100)"),
+    ('302801', "BVI Satu-Mare (ROCJ7810)"),
+    ('302901', "BVF Halmeu (ROCJ4310)"),
+    ('302902', "BVF Aeroport Satu Mare (ROCJ7830)"),
+    ('312801', "BVI Zalău (ROCJ9700)"),
+    ('322801', "BVI Sibiu (ROBV7900)"),
+    ('322901', "BVF Sibiu Aeroport (ROBV7910)"),
+    ('332801', "BVI Suceava (ROIS8230)"),
+    ('332901', "BVF Dorneşti (ROIS2700)"),
+    ('332902', "BVF Siret (ROIS8200)"),
+    ('332903', "BVF Suceava Aero (ROIS8250)"),
+    ('332904', "BVF Vicovu De Sus (ROIS9620)"),
+    ('342801', "BVI Alexandria (ROCR0310)"),
+    ('342901', "BVF Turnu Măgurele (ROCR9100)"),
+    ('342902', "BVF Zimnicea (ROCR5800)"),
+    ('352802', "BVI Timişoara Bază (ROTM8720)"),
+    ('352901', "BVF Jimbolia (ROTM5010)"),
+    ('352902', "BVF Moraviţa (ROTM5510)"),
+    ('352903', "BVF Timişoara Aeroport (ROTM8730)"),
+    ('362901', "BVF Sulina (ROCT8300)"),
+    ('362902', "BVF Aeroport Delta Dunării Tulcea (ROGL8910)"),
+    ('362903', "BVF Tulcea (ROGL8900)"),
+    ('362904', "BVF Isaccea (ROGL8920)"),
+    ('372801', "BVI Vaslui (ROIS9610)"),
+    ('372901', "BVF Fălciu (-)"),
+    ('372902', "BVF Albiţa (ROIS0100)"),
+    ('382801', "BVI Râmnicu Vâlcea (ROCR7700)"),
+    ('392801', "BVI Focșani (ROGL3600)"),
+    ('402801', "BVI Bucureşti Poştă (ROBU1380)"),
+    ('402802', "BVI Târguri și Expoziții (ROBU1400)"),
+    ('402901', "BVF Băneasa (ROBU1040)"),
+    ('512801', "BVI Călăraşi (ROCT1710)"),
+    ('522801', "BVI Giurgiu (ROBU3910)"),
+    ('522901', "BVF Zona Liberă Giurgiu (ROBU3980)"),
+]
+
+STATE_CODES = {
+    'ab': '1',
+    'ar': '2',
+    'ag': '3',
```

Comment:
I see that in the XML you use them like
`STATE_CODES[interface.l10n_ro_edi_stock_start_state_id.code.lower()]`

Why don't you just make them in uppercase, so that you don't have to convert them?